build-tools/packages/build-cli/src/lib/dangerfile.ts

```diff
@@ -24,7 +24,7 @@ const adoConstants = {
 };
 
 const localReportPath = "./artifacts/bundleAnalysis";
-let logger: Logger;
+const logger: Logger = defaultLogger;
```

Comment:
As mentioned above this should be settable. Otherwise callers can't use their own logger.

Reply:
But it needs to be passed in from somewhere so the signatures of the functions may need to change, etc.